Code/GraphMol/DetermineBonds/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+link_directories(${CMAKE_CURRENT_SOURCE_DIR}/../../../External/YAeHMOP/src/yaehmop_project-build)
```

Comment:
It's probably a good idea to put in a comment like this here so that we remember to clean this up at some point:
```
# FIX: there's probably a better way to do this
```